### PR TITLE
Add remaining weapon property runes from TV and LO:KoL

### PR DESCRIPTION
**Pull request by stwlam**
Also add means of adjusting weapon traits from property runes

### CODE REVIEW

File: src/module/item/physical/runes.ts [hunk 1]
```diff
@@ -1125,7 +1184,11 @@ function getPropertyRuneDice(runes: WeaponPropertyRuneType[]): DamageDicePF2e[]
     });
 }
 
-function getPropertyRuneAdjustments(runes: WeaponPropertyRuneType[]): ModifierAdjustment[] {
+function getProperyRuneStrikeAjustments(runes: WeaponPropertyRuneType[]): StrikeAdjustment[] {
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1187]: getPropertyRuneStrikeAjustments

File: src/module/item/physical/runes.ts [hunk 2]
```diff
@@ -1172,13 +1235,14 @@ const WEAPON_VALUATION_DATA: WeaponValuationData = {
...
 
 export {
-    RuneValuationData,
-    WEAPON_VALUATION_DATA,
-    WeaponPropertyRuneData,
-    getPropertyRuneAdjustments,
     getPropertyRuneDice,
     getPropertyRunes,
     getPropertySlots,
+    getProperyRuneModifierAjustments,
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1241]: getPropertyRuneModifierAjustments
getProperyRuneStrikeAjustmentst

File: src/module/actor/character/document.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ import {
 import { ActionTrait } from "@item/action/data.ts";
 import { ARMOR_CATEGORIES } from "@item/armor/values.ts";
 import { ItemType, PhysicalItemSource } from "@item/data/index.ts";
-import { getResilientBonus } from "@item/physical/runes.ts";
+import { getPropertyRuneStrikeAjustments, getResilientBonus } from "@item/physical/runes.ts";
```
Comment (simonward) [inline src/module/actor/character/document.ts:50]:
```suggestion
import { getPropertyRuneStrikeAdjustments, getResilientBonus } from "@item/physical/runes.ts";
```

File: src/module/actor/character/document.ts [hunk 4]
```diff
@@ -1186,7 +1186,11 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
         const ammos = options.ammos ?? [];
 
         // Apply strike adjustments affecting the weapon
-        for (const adjustment of synthetics.strikeAdjustments) {
+        const strikeAdjustments = [
+            synthetics.strikeAdjustments,
+            getPropertyRuneStrikeAjustments(weapon.system.runes.property),
```
Comment (simonward) [inline src/module/actor/character/document.ts:1191]:
```suggestion
            getPropertyRuneStrikeAdjustments(weapon.system.runes.property),
```

File: src/module/item/physical/runes.ts [hunk 5]
```diff
@@ -1125,7 +1184,11 @@ function getPropertyRuneDice(runes: WeaponPropertyRuneType[]): DamageDicePF2e[]
     });
 }
 
-function getPropertyRuneAdjustments(runes: WeaponPropertyRuneType[]): ModifierAdjustment[] {
+function getPropertyRuneStrikeAjustments(runes: WeaponPropertyRuneType[]): StrikeAdjustment[] {
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1187]:
```suggestion
function getPropertyRuneStrikeAdjustments(runes: WeaponPropertyRuneType[]): StrikeAdjustment[] {
```

File: src/module/item/physical/runes.ts [hunk 6]
```diff
@@ -1125,7 +1184,11 @@ function getPropertyRuneDice(runes: WeaponPropertyRuneType[]): DamageDicePF2e[]
...
+    return runes.flatMap((rune) => CONFIG.PF2E.runes.weapon.property[rune].strikeAdjustments ?? []);
+}
+
+function getPropertyRuneModifierAjustments(runes: WeaponPropertyRuneType[]): ModifierAdjustment[] {
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1191]:
```suggestion
function getPropertyRuneModifierAdjustments(runes: WeaponPropertyRuneType[]): ModifierAdjustment[] {
```

File: src/module/item/physical/runes.ts [hunk 7]
```diff
@@ -1172,13 +1235,14 @@ const WEAPON_VALUATION_DATA: WeaponValuationData = {
 };
 
 export {
-    RuneValuationData,
-    WEAPON_VALUATION_DATA,
-    WeaponPropertyRuneData,
-    getPropertyRuneAdjustments,
     getPropertyRuneDice,
+    getPropertyRuneModifierAjustments,
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1239]:
```suggestion
    getPropertyRuneModifierAdjustments,
```

File: src/module/item/physical/runes.ts [hunk 8]
```diff
@@ -1172,13 +1235,14 @@ const WEAPON_VALUATION_DATA: WeaponValuationData = {
...
     getPropertyRunes,
+    getPropertyRuneStrikeAjustments,
```
Comment (simonward) [inline src/module/item/physical/runes.ts:1241]:
```suggestion
    getPropertyRuneStrikeAdjustments,
```

File: src/module/system/damage/weapon.ts [hunk 9]
```diff
@@ -11,7 +11,7 @@ import {
 import { AbilityString } from "@actor/types.ts";
 import { MeleePF2e, WeaponPF2e } from "@item";
 import { NPCAttackDamage } from "@item/melee/data.ts";
-import { getPropertyRuneAdjustments, getPropertyRuneDice } from "@item/physical/runes.ts";
+import { getPropertyRuneDice, getPropertyRuneModifierAjustments } from "@item/physical/runes.ts";
```
Comment (simonward) [inline src/module/system/damage/weapon.ts:14]:
```suggestion
import { getPropertyRuneDice, getPropertyRuneModifierAdjustments } from "@item/physical/runes.ts";
```

File: src/module/system/damage/weapon.ts [hunk 10]
```diff
@@ -356,7 +356,7 @@ class WeaponDamagePF2e {
         // Property Runes
         const propertyRunes = weapon.isOfType("weapon") ? weapon.system.runes.property : [];
         damageDice.push(...getPropertyRuneDice(propertyRunes));
-        const propertyRuneAdjustments = getPropertyRuneAdjustments(propertyRunes);
+        const propertyRuneAdjustments = getPropertyRuneModifierAjustments(propertyRunes);
```
Comment (simonward) [inline src/module/system/damage/weapon.ts:359]:
```suggestion
        const propertyRuneAdjustments = getPropertyRuneModifierAdjustments(propertyRunes);
```

**Review by TikaelSol — APPROVED** [review]

**Pull request merged**